RELEASE-NOTES.txt

```diff
@@ -9,6 +9,9 @@
 * [*] [internal] Change how a post is fetched when selecting a post in Spotlight search result. [#21439]
 * [*] [internal] Change how a post's revision is fetched from the post history screen. [#21440]
 * [**] [internal] Replace the progress indicator implementation in uploading featured image from "Post Settings" [#21438]
+* [**] Block Editor:  Add basic support to view, relocate, and remove the Jetpack Paywall block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/6076]
```

Comment:
I understand this would only affect the Jetpack app, so I'd add the `[Jetpack-only]` tag to the entry. Additionally, I noticed there's an extra space that could be removed.

```suggestion
* [**] [Jetpack-only] Block Editor: Add basic support to view, relocate, and remove the Jetpack Paywall block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/6076]
```

Reply:
Additionally, we'd also need to solve the file conflicts in the release notes file.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/21476/commits/b2f9c89430f58cea98400068b119bafc5d8512fb.